doc/requirements.txt

```diff
@@ -1,4 +1,3 @@
 Sphinx==4.5.0
 sphinx-reredirects<1
 furo==2022.4.7
```

Comment:
We need the `-e .` for RTD as it needs access to `pylint`. We provide `doc/requirements.tx` while we're `cd`'ed into `root`.

I know I looked into this previously but couldn't figure out an elegant solution of allowing both RTD and `cd docs` + `pip install -r ...` to work.

Reply:
How about we suggest to install from the root with ``doc/requirements.txt``, and we remove the requirement from the makefile ? (I.e. it's the job of the prepare-base job to install the proper dependencies.)

Reply:
Can we `cd` in a makefile? That way we could `cd` before and after installing the documentation dependencies.

I do think it would be nice to have one command you can run to completely make the documentation without any further trouble.

Reply:
[21f099a](https://github.com/PyCQA/pylint/pull/6623/commits/21f099a0195fa165040b20c437704ac733b8ff7b)